rudaux/flows.py

```diff
@@ -177,8 +192,9 @@ def build_autoext_flows(config, args):
                 # Fill in submission deadlines
                 submission_sets = subm.build_submission_set.map(unmapped(config), submission_sets)
 
-                # Compute override updates
-                overrides = subm.get_latereg_overrides.map(unmapped(config.latereg_extension_days[group]), submission_sets)
+                # Compute override updates if registration deadline has not passed.
+                if plm.now().in_timezone(config.notify_timezone).timestamp() < plm.from_format(config.registration_deadline, 'YYYY-MM-DD').timestamp():
```

Comment:
This code won't work. Basically this code is run *once* to construct the flow. Using a conditional statement like this when the flow is constructed does not mean the conditional statement is used when the flow is actually run. 

You're better off editing the code inside `subm.get_latereg_overrides`

Reply:
oh right!